lib/licenses.nix

```diff
@@ -740,7 +740,7 @@ lib.mapAttrs (n: v: v // { shortName = n; }) {
 
   xfig = {
     fullName = "xfig";
-    url = "http://mcj.sourceforge.net/authors.html#xfig";
+    url = "http://mcj.sourceforge.net/authors.html#xfig"; # http is broken
```

Comment:
```suggestion
    url = "http://mcj.sourceforge.net/authors.html#xfig"; # https is broken
```

Reply:
Good one! :-) Addressed in 2c45fa52a42c50cf3cd2dcc6483d2ed19d733025